JabMapTestData/dummydata-1.bib

```diff
@@ -0,0 +1,63 @@
+% Encoding: UTF-8
+@INPROCEEDINGS{aPaper,
+	author={T. {Kim} and S. {Park} and T. {Lee}},
```

Comment:
No need to a add braces around last names (in case they consists of single words)